neuromation/api/abc.py

```diff
@@ -117,3 +128,13 @@ def push(self, data: ImageProgressPush) -> None:
     @abc.abstractmethod
     def step(self, data: ImageProgressStep) -> None:
         pass  # pragma: no cover
+
+
+class AbstractDockerContainerProgress(abc.ABC):
```

Comment:
`docker-commit` progress is now separated from `docker-push/pull`. Considering that there's only 2 messages to print all of format `{"status": "message..."}`, we can remove this abstraction.

Reply:
maybe, we can change this class to have two methods: 
- `start_commit()`, and
- `finish_commit()`

Reply:
no the case anymore